packages/cubejs-query-orchestrator/src/driver/BaseDriver.js

```diff
@@ -179,6 +179,13 @@ export class BaseDriver {
     throw new Error('Not implemented');
   }
 
+  /**
+   * @public
+   */
+  onCreate() {
```

Comment:
I didn't find a way where I can run some logic after creation of the driver, so this kinda hook looks like a reasonable thing